apps/app-article/src/components/article-content.js

```diff
@@ -17,8 +18,8 @@ class Content extends Component {
                 return this.renderImage(block, key);
             } else if (blockType === 'html') {
                 return this.renderHtml(block, key);
-            } else if (blockType === 'factbox') {
```

Comment:
Let's actually not remove this entirely, as it will break cached articles for instance. We could modify the object a bit, and pass it to `renderGenericBox`.